### DISCUSSION TITLE
Separating CDN links from direct links جداسازی لینک‌‌های سی‌دی‌ان از دایرکت

### DISCUSSION THREAD

**Discussion by lymanjre** [hiddify/Hiddify-Manager#143] [Ideas ایدهها]
بهتر هست که لینکهای دامینی که از cdn استفاده میکند از لینکهای دامین دایرکت جدا شود. یعنی در کانفیگهای cdn لینکهای دایرکت وجود نداشته باشد
و اگر قسمتی هم برای همه لینکها باشه که عالی میشه

**Comment by hiddify-com** [comment]
الان انجام شده که

**Reply by lymanjre** [reply]
بله. تشکر از شما

**Reply by ghost** [reply]
> الان انجام شده که

  عزیز اگر امکانش باشه این قابلیت طوری بتونی لحاظ کنی برای یوزر بشه ست کرد که کدوم پروتکل ها رو در اختیار داشه باشهhiddify
